cfgov/jinja2/v1/_includes/organisms/filterable-list-controls.html

```diff
@@ -42,102 +44,114 @@
 {% macro _render_filter_fields(controls, form, index) -%}
     {% if controls.title %}
         {% set field_id = 'filter' ~ index ~ '_title' %}
-        <div class="form-l_col
-                    form-l_col-1">
-            <div class="form-group">
-                <label class="form-label-header"
-                       for="{{ field_id }}">
-                    Item name
-                </label>
-                {{ form.render_with_id(form.title, field_id) }}
+        <div class="content-l_col
+                    content-l_col-1">
+            <div class="o-form_group">
+                <fieldset class="o-form_fieldset">
+                    <label class="a-label a-label__heading"
+                           for="{{ field_id }}">
+                        Item name
+                    </label>
+                    {{ form.render_with_id(form.title, field_id) }}
+                </fieldset>
             </div>
         </div>
     {% endif %}
     {% if controls.categories.filter_category %}
-        <div class="form-l_col
-                    form-l_col-1-3">
-            <div class="form-group">
-                <label class="form-label-header"
-                       for="categories">
-                    {% if 'leadership-calendar' in controls.categories.page_type %}
-                        Calendars
-                        </label>
-                        {% for slug, name in choices_for_page_type(controls.categories.page_type) %}
-                            {{ _filter_selectable('radio', index, category_label(slug), 'categories_' ~ slug, 'categories', slug) }}
-                        {% endfor %}
-                    {% else %}
-                        Category
-                        </label>
-                        {% for slug, name in choices_for_page_type(controls.categories.page_type) %}
-                            {{ _filter_selectable('checkbox', index, category_label(slug), 'categories_' ~ slug, 'categories', slug) }}
-                        {% endfor %}
-                    {% endif %}
+        <div class="content-l_col
```

Comment:
This file should be printed on paper and burned.  That isn't your problem though.